Cargo.toml

```diff
@@ -18,80 +18,100 @@ members = ["crates/*/"]
+nu_plugin_s3 = { version = "0.25.2", path = "./crates/nu_plugin_s3", optional = true }
+nu_plugin_selector = { version = "0.25.2", path = "./crates/nu_plugin_selector", optional = true }
+nu_plugin_start = { version = "0.25.2", path = "./crates/nu_plugin_start", optional = true }
+nu_plugin_sys = { version = "0.25.2", path = "./crates/nu_plugin_sys", optional = true }
+nu_plugin_textview = { version = "0.25.2", path = "./crates/nu_plugin_textview", optional = true }
+nu_plugin_to_bson = { version = "0.25.2", path = "./crates/nu_plugin_to_bson", optional = true }
+nu_plugin_to_sqlite = { version = "0.25.2", path = "./crates/nu_plugin_to_sqlite", optional = true }
+nu_plugin_tree = { version = "0.25.2", path = "./crates/nu_plugin_tree", optional = true }
+nu_plugin_xpath = { version = "0.25.2", path = "./crates/nu_plugin_xpath", optional = true }
 
 # Required to bootstrap the main binary
 clap = "2.33.3"
-ctrlc = {version = "3.1.6", optional = true}
-futures = {version = "0.3.5", features = ["compat", "io-compat"]}
+ctrlc = { version = "3.1.6", optional = true }
+futures = { version = "0.3.5", features = ["compat", "io-compat"] }
 itertools = "0.10.0"
 log = "0.4.11"
 pretty_env_logger = "0.4.0"
 
 [dev-dependencies]
 dunce = "1.0.1"
-nu-test-support = {version = "0.25.2", path = "./crates/nu-test-support"}
+nu-test-support = { version = "0.25.2", path = "./crates/nu-test-support" }
 
 [build-dependencies]
 
 [features]
 ctrlc-support = ["nu-cli/ctrlc", "nu-command/ctrlc"]
-directories-support = ["nu-cli/directories", "nu-cli/dirs", "nu-command/directories", "nu-command/dirs", "nu-data/directories", "nu-data/dirs", "nu-engine/dirs"]
+directories-support = [
+    "nu-cli/directories",
+    "nu-cli/dirs",
+    "nu-command/directories",
+    "nu-command/dirs",
+    "nu-data/directories",
+    "nu-data/dirs",
+    "nu-engine/dirs",
+]
 ptree-support = ["nu-cli/ptree", "nu-command/ptree"]
 rustyline-support = ["nu-cli/rustyline-support", "nu-command/rustyline-support"]
 term-support = ["nu-cli/term", "nu-command/term"]
 uuid-support = ["nu-cli/uuid_crate", "nu-command/uuid_crate"]
-which-support = ["nu-cli/ichwh", "nu-cli/which", "nu-command/ichwh", "nu-command/which"]
+which-support = [
+    "nu-cli/ichwh",
+    "nu-cli/which",
+    "nu-command/ichwh",
+    "nu-command/which",
+]
 
 default = [
-  "sys",
-  "ps",
-  "textview",
-  "inc",
-  "directories-support",
-  "ctrlc-support",
-  "which-support",
-  "ptree-support",
-  "term-support",
-  "uuid-support",
-  "rustyline-support",
-  "match",
-  "post",
-  "fetch",
-  "zip-support",
+    "sys",
+    "ps",
+    "textview",
+    "inc",
+    "directories-support",
+    "ctrlc-support",
+    "which-support",
+    "ptree-support",
+    "term-support",
+    "uuid-support",
+    "rustyline-support",
+    "match",
+    "post",
+    "fetch",
+    "zip-support",
 ]
-extra = ["default", "binaryview", "tree", "clipboard-cli", "trash-support", "start", "bson", "sqlite", "s3", "chart", "xpath", "selector"]
```

Comment:
Did you mean to remove the `extra` feature here? This is how we enable all the optional features.